notebooks/post_DL3_analysis.ipynb

```diff
@@ -1,36 +1,58 @@
 {
```

Comment:
<p>As of Gammapy 1.1 models can be stored serialized in YAML format, no need to longer rely on pickle to write read the models.</p>

<pre class="ql-syntax" spellcheck="false">stacked_dataset.models.write("sed_model.yml")
</pre><p>which will also produce the "sed_model_covariance.yml" file at the same time.</p><p>To read it back:</p>

<pre class="ql-syntax" spellcheck="false">from gammapy.modeling.models import Models

# Read back stored best-fit model
best_model = Models.read("sed_model.yml")
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/1262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='85'/>